kumascript/macros/htmlattrdef.ejs

```diff
@@ -1,2 +1,2 @@
 <%/* Example: <span class="script">htmlattrdef("placeholder")</span> */%>
-<strong id="attr-<%=$0%>"><code><%=$0%></code></strong>
+<a id="attr-<%=$0%>" href="#attr-<%=$0%>"><strong><code><%=$0%></code></strong></a>
```

Comment:
Not introduced here but seeing that we are making changes. Let’s not use the `strong` element here. The ideal would be if we could give these description lists a class and then style the `dt` element but:

1. Not sure how feasible that is
2. Not sure what will happen with description lists in the markdown future :)

Therefore, I am thinking let’s switch to using the `b` tag. The `strong` element is meant to convey importance, seriousness, or urgency, and here we are really just using it to ensure they are rendered in a bold font style to draw attention to them.

Reply:
> The strong element is meant to convey importance, seriousness, or urgency
> using it to ensure they are rendered in a bold font style to draw attention to them

Well, I might be missing something here, but isn't the `<strong>` tag meant to highlight an important part of the document to draw attention to it?

Reply:
Well, it is sometimes really hard to nail down :) In this case, I am thinking that the content is already semantically marked up as a description list, and the item we are using as the link is defined as a term that we then describe in the child `dd` element. The reason it is bold is mainly that we want to further visually distinguish the term from its description so, the `b` element should be sufficient.

Reply:
OK, thanks for the explanation. I will make the changes.